### PR TITLE
Add support for poppler's hide-annotations flag

### LINKED ISSUE #17: Ability to disable link/ref highlighting
Greetings! 

Pdftoppm/poppler has this weird "feature" to highlight hyperlinks with very bright rectangles that are never rendered in mainstream pdf readers and is very annoying. 

I was looking for an appropriate flag to disable it and fork/merge your repo but in vain. I traced an issue in [this conversation](https://markmail.org/message/2sbcuvp4aerhc3go) but doesn't seem to offer any solution in the end. I also started digging into [pdftoppm.cc source code](https://github.com/justmoon/poppler-http/blob/master/utils/pdftoppm.cc) but nothing caught my eye yet. I was wondering if you guys had any idea how to fix the issue before I continue digging into the source code.

This is general issue and happens to every arxiv.org pdf I have tried. Attaching sample bellow anyways.

Example file: page-15.png
PDF Source: https://arxiv.org/abs/1806.00451 
page-15

### PR DESCRIPTION
**Pull request by StaticRocket**
Resolves #17 

Adds support for poppler's hide-annotations flag which removes the drawn bounding boxes for links and other annotations.

### CODE REVIEW

**Comment by Belval** [comment]
Thank you for implementing this, do you think you could add a test to ensure that it works?

**Comment by StaticRocket** [comment]
Yeah, I'll add a test and update the docs tonight.

Could you tell me if raising a not implemented error to inform the user that pdftocario doesn't support the hide annotations arguments is the right call?

**Comment by Belval** [comment]
Yeah it works for me. There seems to be some effort by the main developers to implement it but in the meantime we can just raise an exception.

File: pdf2image/pdf2image.py [hunk 1]
```diff
@@ -170,9 +174,12 @@ def convert_from_path(
             single_file,
             grayscale,
             size,
+            hide_annotations,
         )
 
         if use_pdfcairo:
+            if hide_annotations:
+                raise NotImplementedError("Hide annotations flag not implemented in pdftocario.")
```
Comment (Belval) [inline pdf2image/pdf2image.py:182]: Nit: typo s/pdftocario/pdftocairo

**Review by Belval — APPROVED** [review]

**Comment by Belval** [comment]
I'll test it today or tomorrow and merge it if everything works as expected. Thank you for your contribution!

**Comment by Belval** [comment]
Thank you for your contribution.

**Pull request merged**